build/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.14 as build
 LABEL stage=build
 
+# Disable cgo as it breaks containerd builds
+ENV CGO_ENABLED=false
```

Comment:
Use `CGO_ENABLED=0` instead.

Reply:
Now that you are doing it here, you can remove `CGO_ENABLED=0` under the CNI plugins.